sbt-plugin-test/build.sbt

```diff
@@ -125,36 +136,25 @@ lazy val multiTest = crossProject.
       }, "fullOptJS does not have the correct scalaJSSourceMap attribute")
     }
   ).
-  jvmSettings(versionSettings: _*).
-  jvmSettings(
+  dependsOn(testFrameworkJS % "test")
+
+lazy val multiTestJVM = project.in(file("multiTest/jvm")).
+  settings(multiTestCommonSettings).
+  settings(versionSettings: _*).
+  settings(
     name := "Multi test framework test JVM",
     libraryDependencies +=
       "com.novocode" % "junit-interface" % "0.9" % "test",
 
-    // Test crossPlatform (as a setting, it's evaluated when loading the build)
-    crossPlatform ~= { value =>
-      assert(value == JVMPlatform,
-          "crossPlatform should be JVMPlatform in multiTestJVM")
+    // Test platformDepsCrossVersion (as a setting, it's evaluated when loading the build)
+    platformDepsCrossVersion := {
+      val value = platformDepsCrossVersion.value
+      if (!sbtVersion.value.startsWith("0.")) {
+        // In 0.13, CrossVersions do not have a meaningful ==, but they do in 1.0
+        assert(value == CrossVersion.binary,
+            "platformDepsCrossVersion should be CrossVersion.binary in multiTestJVM")
+      }
```

Comment:
This too, no? Or what is this testing?

Reply:
This one is mostly by symmetry with the other one. Arguably it's also testing that the setup of `platformDepsCrossVersion` in `ScalaJSPlugin` does not affect other projects (like, if we erroneously put it in `override def globalSettings` instead of `projectSettings`).